playwright.config.ts

```diff
@@ -39,9 +39,9 @@ const config: PlaywrightTestConfig = {
   },
   projects: [
     {
-      name: 'chromium',
+      name: 'webkit',
       use: {
-        ...devices['Desktop Chrome'],
+        ...devices['Desktop Safari'],
```

Comment:
Think we don't want to switch to `webkit`.
Also, this version of `Safari` does not have `ApplePay` inside.

Reply:
Thanks for the find, will update